### ISSUE TITLE
Can I Analyse Big Binary In Radare2?

### ISSUE THREAD

**Issue by mendax47** [radare/radare2#7868]
I Am Having Problem When I Try To Analyse A Big Binary (Tested At 37 MB And 107 MB)....In Radare2 The Function Doesn't Show.... I Am Giving You Just An Example.... 

Please Have A Look
http://imgur.com/a/x12qF

**Comment by radare** [comment]
i dont think you have analyzed anything to get this output. yes you can analyze, but it will take a while for such file sizes

**Comment by Maijin** [comment]
Probably super old r2 too.

http://bin.rada.re/radare2-w32-1.6.0-git.zip + https://conemu.github.io/en/Screenshots.html

**Comment by mendax47** [comment]
Nope I Am Using 1.6.0.. But Method Doesn't Show At All...

**Comment by mendax47** [comment]
@radare yah.. i know but why the function doesn't show up....?

**Comment by radare** [comment]
which commands did you typed before reaching that screen? without knowing that or having the file i cant say..

**Comment by mendax47** [comment]
@radare 
radare2 bin-name
then aa
then aaa

**Comment by radare** [comment]
Type e anal.hasnext=true before aa and try again

**Comment by mendax47** [comment]
@radare radare2 Stop Working... Crash...

**Comment by radare** [comment]
probably because enomem probably, but the functions are there, maybe a backtrace may help to understand that, just “stop working crash” doesnt describes the real issue at all

**Comment by Maijin** [comment]
can you share the bin/hash ?

**Comment by radare** [comment]
isnt he asking for analyzing 100MB binaries?

**Comment by mendax47** [comment]
Install visual studio code.. then try to open the Code.exe 54.4 mb
hash: 7981C8A9FA6507D26C9FBB33AEE75E20AF204968D9D375E5CDAC222BE0075774

**Comment by Nanreit** [comment]
Isn't the current version 1.5.0?

**Comment by radare** [comment]
No

**Comment by mendax47** [comment]
Update: The Problem Is Only For PE File... Dynamic Link Library (DLL) Is Fine Here...

**Comment by Maijin** [comment]
[7981c8a9fa6507d26c9fbb33aee75e20af204968d9d375e5cdac222be0075774.zip](https://github.com/radare/radare2/files/1136537/7981c8a9fa6507d26c9fbb33aee75e20af204968d9d375e5cdac222be0075774.zip)

**Comment by mendax47** [comment]
@Maijin This Is Not The File I Mentioned Earlier. The Program Was Visual Studio Code....

**Comment by Maijin** [comment]
That's the hash you sent me..

**Comment by Maijin** [comment]
Just so you know It is zip encrypted with "infected" password.

**Comment by radare** [comment]
can we close this?
